.github/workflows/README.md

```diff
@@ -0,0 +1,27 @@
+# Workflow Management
+
+Here we have the start of a system to automatically generated workflows (currently only for transforms). 
+In general, the design is to use templates and `make` to generate/update the workflows.
+
+```
+make
+git commit -a -s -m "update workflows"
+git push
+```
+
+should be sufficient.
+
+## Transforms 
+We define a unique test workflow for each transform, based on a common template [test-transform.template](test-transform.template).
```

Comment:
Just for the testing its worth to add a small change to the data-prep-lib code in this PR and see to all the runs in parallel...

Reply:
Can you explain at bit more?  As I've seen, all these transform tests do run in parallel (when a lib change is made).